processing/src/main/java/org/apache/druid/segment/virtual/MultiValueExpressionDimensionSelector.java

```diff
@@ -50,11 +50,13 @@ public MultiValueExpressionDimensionSelector(ColumnValueSelector<ExprEval> baseS
     this.baseSelector = baseSelector;
   }
 
+  @Nullable
```

Comment:
Does all callers need to check the returned eval? Seems there are a couple of missing places.

Reply:
I'd like to do this in phases, there are about 600+ issues that intelliJ reports in the druid-processing sub module. I'll pick through more of them in my next PR

Reply:
Oh I meant for this method. This method is technically private and only used in this class. Some of the callers are still not checking nulls, for example [here](https://github.com/apache/druid/pull/9760/files/79eb23832004e90ed498aa1448d3bc9513cd6b50#diff-2ae123ea2c178ff9151ddbb31ed5f2b8R108).

Reply:
These are checked on the hot loop. I've removed them for now, it's hard to reason whether or not it's possible to return null here. I think it can't, but I'm not 100% sure so I'm reverting this change and the associated null check

Reply:
Oh yeah, it doesn't seem possible to return null since `ColumnValueSelector` returns an `ExprEval`.